packages/eslint-plugin/tests/rules/dot-notation.test.ts

```diff
@@ -239,8 +239,9 @@ x.pub_prop = 123;
       errors: [{ messageId: 'useDot', data: { key: q('SHOUT_CASE') } }],
     },
     {
-      code: 'a\n' + "  ['SHOUT_CASE'];",
-      output: 'a\n' + '  .SHOUT_CASE;',
+      // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
+      code: "a\n  ['SHOUT_CASE'];",
```

Comment:
I think you should be able to use the noFormat tagged template rather than having to disable the rule with a comment

```suggestion
      code: noFormat`a\n  ['SHOUT_CASE'];`,
```

Here and elsewhere

Reply:
Consider making it mostly formatted, though, like 
```suggestion
      code: "
a
['SHOUT_CASE'];
      ",
```